src/main/java/jenkins/plugins/logstash/persistence/BuildData.java

```diff
@@ -152,6 +152,11 @@ public BuildData(AbstractBuild<?, ?> build, Date currentTime) {
         }
       }
     }
+    try {
+      buildVariables.putAll(build.getEnvironment(listener));
+    } catch (Exception e) {
```

Comment:
I was warned of potential unhanded exceptions (`IOException` & `InterruptedException`) generated by `build.getEnvironment(listener)` but I admit I'm not sure exactly what condition generate that exception after looking at `public EnvVars getEnvironment` so I wasn't savvy enough to describe the exception in the catch.

I was sticking to the same as what the pipeline job type code does to load Envs into buildVariables: https://github.com/diginc/logstash-plugin/blob/5fb2d13177e68aed2f39e49aa6006848eae3875f/src/main/java/jenkins/plugins/logstash/persistence/BuildData.java#L180